redpanda/templates/statefulset.yaml

```diff
@@ -70,11 +70,12 @@ spec:
               rpk --config $CONFIG config set redpanda.advertised_rpc_api.address $SERVICE_NAME;
               rpk --config $CONFIG config set redpanda.advertised_rpc_api.port {{ .Values.config.redpanda.rpc_server.port }};
               {{- if .Values.loadBalancer.enabled }}
-              rpk --config $CONFIG config set redpanda.advertised_kafka_api.address {{ template "redpanda.name" . }}-$NODE_ID.{{.Values.loadBalancer.parentZone}}
+              ADVERTISED_KAFKA_API='[{"address": "'"$SERVICE_NAME"'", "port": {{.Values.config.redpanda.kafka_api.port}}, "name":"internal"},{"address": "{{ template "redpanda.fullname" . }}-'"$NODE_ID"'.{{.Values.loadBalancer.parentZone}}", "port": {{.Values.loadBalancer.port}}, "name":"external"}]';
```

Comment:
For some reason when running in the cluster, rpk only renders:
```yaml
  advertised_kafka_api:
  - address: redpanda-0.redpanda.redpanda.svc.cluster.local
```
But if I run the command locally, I get:
```yaml
  advertised_kafka_api:
  - address: redpanda-0.redpanda.redpanda.svc.cluster.local
    name: internal
    port: 9092
  - address: redpanda-0.my-project.gcp.domain.com.
    name: external
    port: 9092
```

Reply:
@dotnwat  ^^

Reply:
Ignore this, it's now fine. rpk start doesn't understand it, though.